Sources/ProjectSpec/Scheme.swift

```diff
@@ -24,12 +24,12 @@ public struct Scheme: Equatable {
         self.archive = archive
     }
 
-    public init(name: String, targets: [BuildTarget], debugConfig: String, releaseConfig: String, gatherCoverageData: Bool = false, commandLineArguments: [String: Bool] = [:]) {
+    public init(name: String, targets: [BuildTarget], debugConfig: String, releaseConfig: String, gatherCoverageData: Bool = false, commandLineArguments: [String: Bool] = [:], testables: [String] = []) {
```

Comment:
Looking at this now, I think I will change this init to take a `name: String, targetScheme: TargetScheme, debugConfig: String, releaseConfig: String` and move as an extension in the ProjectGenerator.
But that can happen after this.